src/client/runner/automation/utils/get-element.js

```diff
@@ -0,0 +1,73 @@
+import hammerhead from '../../deps/hammerhead';
+import testCafeCore from '../../deps/testcafe-core';
+import Cursor from '../cursor';
```

Comment:
@inikulin `Cursor` is a static class in fact (https://github.com/DevExpress/testcafe/pull/296/files#diff-1784228fd59746f9c576f50186d080ddR13)
Should we import it as an object everywhere (`import cursor from '../cursor';`) ? The same with CursorUI and IframeCursorUI